services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/config/env/AppConfig.java

```diff
@@ -14,20 +14,22 @@
         limitations under the License.
 */
 
-package com.futurewei.alcor.controller.app.onebox;
+package com.futurewei.alcor.dataplane.config.env;
```

Comment:
Please don't duplicate legacy codes. This class basically shouldn't be used at all in any microservices including DP manager.

Reply:
some basic constants I could move it to classes who use them or just remove them totally, please refer to 136f4b5